README.md

```diff
@@ -77,13 +77,14 @@ The following CMake configuration options are supported.
 | CMAKE_BUILD_TYPE     | Release        |            | With full optimizations.                   |
 |                      | Debug          |            | With debug symbols.                        |
 |                      | RelWithDebInfo |            | With full optimizations and debug symbols. |
-| CMAKE_INSTALL_PREFIX | <string>       | /usr/local | The directory to install the library in.   |
+|                      | Dev/DevDebug   |            |                                            |
```

Comment:
Let's split these into two lines (like with `Release` and `Debug` above), and include descriptions (e.g. `With warnings treated as errors and full optimizations`)

Reply:
Looks good, except `DevDebug` doesn't have full optimizations.